### PR TITLE
38 save button to business cards

### PR DESCRIPTION
**Pull request by GeorgeSwift16**

### CODE REVIEW

**Review by DanForder — CHANGES_REQUESTED** [review]
fix merge conflicts

**Comment by GeorgeSwift16** [comment]
merged main, checked tests, checked build - ready

File: src/components/Button/Button.scss [hunk 1]
```diff
@@ -42,7 +42,18 @@
...
+.button--details,
+.button--Remove,
+.button--Save {
+  margin: 0 0.2rem;
+  width: 7rem;
+}
+.button--Save {
+  background-color: $orange;
+}
+.button--Remove {
```
Comment (DanForder) [inline src/components/Button/Button.scss:54]: lower-case-hypenated classnames

File: src/components/Card/Card.jsx [hunk 2]
```diff
@@ -26,8 +26,20 @@ const Card = ({
 
     return shortenedDescription;
   };
+
   const cardClassName = page ? `card card--${page}` : "card";
+  const getButtonType = (page) => {
+    let buttonName = "";
+    if (page === "home") {
+      buttonName = "Save";
```
Comment (DanForder) [inline src/components/Card/Card.jsx:34]: *save

File: src/components/Card/Card.jsx [hunk 3]
```diff
@@ -26,8 +26,20 @@ const Card = ({
...
+    } else {
+      buttonName = "Remove";
```
Comment (DanForder) [inline src/components/Card/Card.jsx:36]: *remove

**Review by DanForder — CHANGES_REQUESTED** [review]

**Review by DanForder — CHANGES_REQUESTED** [review]
fix

**Review by DanForder — APPROVED** [review]

**Review by DanForder — APPROVED** [review]

**Review by ekauf — APPROVED** [review]

**Pull request merged**